packages/cli/src/config/schema.ts

```diff
@@ -1012,6 +1016,53 @@ export const schema = {
 		},
 	},
 
+	eventBus: {
+		enabled: {
+			doc: 'Whether event logging is enabled.',
+			format: Boolean,
+			default: true,
+			env: 'N8N_EVENTBUS_ENABLED',
+		},
+		checkUnsentInterval: {
+			doc: 'How often (in ms) to check for unsent event messages. Can in rare cases cause a message to be sent twice. 0=disabled',
+			format: Number,
+			default: 0,
+			env: 'N8N_EVENTBUS_CHECKUNSENTINTERVAL',
+		},
+		destinations: {
+			loadAtStart: {
+				doc: 'Whether destinations are loaded from Db and initialized at startup.',
+				format: Boolean,
+				default: true,
+				env: 'N8N_EVENTBUS_DESTINATIONS_STARTUP',
```

Comment:
Why would I choose not to load from bd? I guess once registered I expect n8n to always load this automatically.

Reply:
same as above. was an early option and will now be removed.